repo/migrations/migration005.go

```diff
@@ -139,6 +140,7 @@ func (Major005) Up(repoPath string, pinCode string, testnet bool) error {
 		}
 	}
 
+	// @todo should this be removed?
```

Comment:
this comment should be removed, as previously discussed, seems reintroduced from the merge